chemprop/args.py

```diff
@@ -523,6 +535,30 @@ def bond_feature_scaling(self) -> bool:
         """
         return not self.no_bond_features_scaling
 
+    @property
+    def atom_constraints(self) -> List[bool]:
+        """
+        A list of booleans indicating whether constraints applied to output of atomic properties.
+        """
+        if self.is_atom_bond_targets and self.constraints_path:
+            header = chemprop.data.utils.get_header(self.constraints_path)
+            atom_constraints = [target in header for target in self.atom_targets]
+        else:
+            atom_constraints = [None] * len(self.atom_targets)
```

Comment:
Why [None] and not [False]? Same at L559.

Reply:
Thanks! It should be `False`.